solutions/03-building-a-streaming-pipeline/src/test/java/marketplace/OrderServiceIntegrationTest.java

```diff
@@ -162,6 +162,7 @@ public void pricesWithTax_shouldReturnTheCorrectPrices() {
     @Test
     @Timeout(60)
     public void createFreeShippingTable_shouldCreateTheTable() {
+        deleteTable(orderQualifiedForFreeShippingTableName);
```

Comment:
VSCode doesn't respect the shutdown hooks that normally delete this table if you terminate the tests prematurely. This can leave things in a state where the test will fail because the table already exists.

Proactively deleting the table prior to running ensures that this doesn't happen.